docker/Dockerfile.nvidia

```diff
@@ -117,6 +117,35 @@ ENV DEVITO_ARCH="nvc"
 ENV DEVITO_PLATFORM="nvidiaX"
 ENV DEVITO_LANGUAGE="openacc"
 
+########################################################################
+# NVC for GPUs via CUDA config
+########################################################################
+FROM sdk-base as nvcc
+
+# Make devito env vars file and extras
+ADD docker/nvdashboard.json /app/nvdashboard.json
+
+# mpi4py
+RUN CFLAGS=-noswitcherror /venv/bin/pip install --no-cache-dir mpi4py && rm -rf ~/.cache/pip
+    # /venv/bin/jupyter lab workspaces import /app/nvdashboard.json
+
+ENV DEVITO_ARCH="cuda"
+ENV DEVITO_PLATFORM="nvidiaX"
+ENV DEVITO_LANGUAGE="cuda"
+
+########################################################################
+# NVC for CPUs config
+########################################################################
+FROM sdk-base as nvc-host
+
+# mpi4py
+RUN CFLAGS=-noswitcherror /venv/bin/pip install --no-cache-dir mpi4py && rm -rf ~/.cache/pip
```

Comment:
That's the cuda aware mpi right? Is it creating issue using it from the host?

Reply:
no, we good